recipes/s100py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "s100py" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+   url: https://github.com/noaa-ocs-s100/s100py/archive/refs/tags/v{{ version }}.tar.gz
+   sha256: 727b87b4d61d9cbe41249862ac1902adab4db3162b6f7c2c9810439697f70d73
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
+    - h5py
+    - numpy
+    - gdal >=3.2
+
+test:
+  requires:
+    - pytest
```

Comment:
You shouldn't need to include `pytest`  unless you are actually running a test suite

Reply:
Thanks, we have removed pytest